fivex/api/__init__.py

```diff
@@ -192,6 +192,7 @@ def region_data_for_region_table(chrom: str, start: int, end: int):
                 "log_pvalue": row.log_pvalue,
                 "beta": row.beta,
                 "stderr_beta": row.stderr_beta,
+                "symbol": row.symbol,
```

Comment:
What is the purpose of this verbose transformation? (vs, say, `dataclassinstance.asdict`)

Reply:
We are only selecting a subset of the data to pass on, as part of an effort to limit the amount of data being transferred. You're correct that this could probably be done more efficiently, so please feel free to address this as you see fit.

Reply:
> We are only selecting a subset of the data to pass on, as part of an effort to limit the amount of data being transferred. You're correct that this could probably be done more efficiently, so please feel free to address this as you see fit.

I've simplified it slightly- the main effect of the change I pushed is to make clear that fields are being extracted (not renamed, transformed, etc). It reduces the amount of reading/ thinking for these 20 lines when maintaining.